src/main/webapp/business.js

```diff
@@ -1,42 +1,70 @@
-// implementing map
+function getInfo() {
+  fetch('/business').then(response => response.json()).then((json) => {
+    var counter = 0;
+    for (var i = 1; i < json.length; i++) {
+      if (json[i][1] == 'in-person') {  // makes list of in-person businessess
```

Comment:
for readability, it would be good to add const for the indexes 
const BUSINESS_TYPE_IDX = 1
similarly for index 0 & 2 below

Reply:
done! great suggestion